lib/components/Search.dart

```diff
@@ -27,35 +36,20 @@ class Search extends StatelessWidget {
         hintText: "Try to find...",
         hintStyle:
             $styles.text.subheading1.copyWith(color: $styles.colors.grey),
-        prefixIcon: SizedBox(
-          width: 20,
-          height: 20,
+        prefixIcon: SizedBox.square(
```

Comment:
SizedBox.square helped reduce a bunch across the app instead of specifying the same height and width